From a7b56b26cfec799a456d2d1b390d0c3127fc65ea Mon Sep 17 00:00:00 2001
From: Piotr Pawliczek <pawliczek@google.com>
Date: Wed, 24 Sep 2025 13:53:22 -0700
Subject: [PATCH 21/56] Do not use BIND_INCLUDE_CAPABILITIES when bind
 PrintService

This is backport of ag/31156800 to 25Q2 security staging branch. This
fix was developed behind the aconfig flag
com.android.server.print.do_not_include_capabilities and landed in
25Q3.

Bug: 291281543
Test: CQ
Cherrypick-From: https://googleplex-android-review.googlesource.com/q/commit:96d05a62683b53f6ba9c83ae43f40f119cd6115b
Merged-In: I2b9d6b68e7aaa71374a6c5e45796c73a1f3abfa0
Change-Id: I2b9d6b68e7aaa71374a6c5e45796c73a1f3abfa0
---
 .../print/java/com/android/server/print/RemotePrintService.java  | 1 -
 1 file changed, 1 deletion(-)

diff --git a/services/print/java/com/android/server/print/RemotePrintService.java b/services/print/java/com/android/server/print/RemotePrintService.java
index b85671581..e841e77f2 100644
--- a/services/print/java/com/android/server/print/RemotePrintService.java
+++ b/services/print/java/com/android/server/print/RemotePrintService.java
@@ -572,7 +572,6 @@ final class RemotePrintService implements DeathRecipient {
 
         boolean wasBound = mContext.bindServiceAsUser(mIntent, mServiceConnection,
                 Context.BIND_AUTO_CREATE | Context.BIND_FOREGROUND_SERVICE
-                        | (Flags.doNotIncludeCapabilities() ? 0 : Context.BIND_INCLUDE_CAPABILITIES)
                         | Context.BIND_ALLOW_INSTANT,
                 new UserHandle(mUserId));
 
-- 
2.52.0

